master.js

```diff
@@ -169,31 +167,31 @@ class AlAzureMaster {
             }],
             callback
         );
-    };
+    }
 
     getAppSettings(callback) {
-        return this._azureWebsiteClient.webApps.listApplicationSettings(
```

Comment:
Is there any specific reason to remove `return` here and below?
The reason it was here is to have at least one return statement per function

Reply:
Nope thats an artefact of my promis implementation. I'll remove it.

Reply:
I mean is JS a return is not strictly required. all functions have an implicit `return undefined` at the end. so there is no danger that the function will not terminate or get GCed. But I'll just put it back in because we're doing that stylistically.